spec/lib/vmdb/appliance_spec.rb

```diff
@@ -20,4 +20,42 @@
       expect(file.read).to include(out)
     end
   end
+
+  describe ".log_config" do
+    before do
+      @logger = double(:info)
+      stub_server_configuration(:api            => {
+                                  :token_ttl              => "10.minutes",
+                                  :authentication_timeout => "30.seconds"},
+                                  :password               => "top_secret",
+                                  :amazon_secret          => "top_secret",
+                                :authentication => {
+                                  :basedn                      => "ou=org,dc=domain,dc=com",
+                                  :bind_dn                     => "cn=name,dc=domain,dc=com",
+                                  :bind_pwd                    => "top_secret",
+                                  :get_direct_groups           => true,
+                                  :group_memberships_max_depth => 2})
+      allow(@logger).to receive(:info)
+    end
+
+    it "does not log password" do
+      expect(@logger).not_to receive(:info).with(/password/)
+      described_class.send(:log_config, :logger => @logger)
```

Comment:
Is log_config a private method?  If so, can you change the describe on line 24 to `describe ".log_config (private)" do` so that it's clear we are testing a private method?